compiler/luci/export/src/SerializedData.h

```diff
@@ -131,8 +132,8 @@ struct SerializedModelData final
    * @param builtin_code
    * @return idx of opcode in table of opcodes (see schema)
    */
-  uint32_t registerBuiltinOpcode(circle::BuiltinOperator builtin_code, const int32_t op_version);
-  uint32_t registerCustomOpcode(const std::string &custom_op);
+  uint32_t registerBuiltinOpcode(circle::BuiltinOperator builtin_code, const int32_t op_version,
+                                 const std::string &custom_code);
```

Comment:
Q) please share why remove `registerCustomOpcode` and merge into `registerBuiltinOpcode`

Reply:
It was because of following reason :)

- To reduce duplicated codes
  - Only the content of  `custom_code` is different for both functions
- Both functions store same structure
  - Both CustomOp and NormalOp should store same data(i.e. `builtin_code`, `op_version`, `custom_code`)
  - CustomOp is not special case, just NormalOp has empty string for `custom_code`